stemgen.py

```diff
@@ -64,7 +63,15 @@
     else os.path.join(PROCESS_DIR, args.OUTPUT_PATH)
 )
 FORMAT = args.FORMAT
-DEVICE = args.DEVICE
+
+# Automatically set DEVICE to "cuda" if CUDA is available, otherwise set it to "cpu"
+DEVICE = args.DEVICE if args.DEVICE is not None else ("cuda" if torch.cuda.is_available() else "cpu")
```

Comment:
💯

Reply:
Yep! Sorry for the delay, got held up at work.